hubblestack/extmods/modules/fdg.py

```diff
@@ -131,7 +131,7 @@
 RETURNER_ID_BLOCK = None
 
 
-def fdg(fdg_file, starting_chained=None):
+def run(fdg_file, starting_chained=None):
```

Comment:
Instead of renaming it to run we should create a new method 'run' to maintain backward compatibility with older fdg profiles

Reply:
As discussed, we are renaming it to run() only